Formula/trinsic.rb

```diff
@@ -0,0 +1,18 @@
+class Trinsic < Formula
+  desc "Command-line interface for Trinsic Ecosystems APIs"
+  homepage "https://docs-v2.trinsic.id/"
+  url "https://github.com/trinsic-id/sdk/releases/download/v1.0.1/cli-1.0.0.tar.gz"
+  sha256 "35f1b17cd80ef52dd8e628380c7401bc1b8e818b63033cce0d5ad328558a71a6"
+  license "Apache-2.0"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    version_text = shell_output "#{bin}/trinsic --version"
+    assert version_text.downcase["trinsic"]
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
@carlocab Of course. Added new one which checks for valid configuration. Would that suffice? I could add few other config related tests, but internally they operate over the config anyway.